foundation/core/variables.tf

```diff
@@ -104,6 +104,55 @@ variable "enable_ms_defender" {
   default     = false
 }
 
+variable "func_identifier" {
+  type    = string
+  default = "billing-alert-func"
+}
+
+variable "slack_webhook_url" {
+  type      = string
+  default   = ""
+  sensitive = true
+}
+
+variable "teams_webhook_url" {
+  type      = string
+  default   = ""
+  sensitive = true
+}
+
+variable "budget_time_start" {
+  type    = string
+  default = ""
+}
+
+variable "subscriptions" {
+  type    = map(any)
+  default = { "default" : "" }
+}
+
+variable "budget_amounts" {
+  type    = map(any)
+  default = { "default" : 1000 }
+}
+
+variable "budget_time_grains" {
+  description = "A choice of time grain, options are: Annually, BillingAnnual, BillingMonth, BillingQuarter, Monthly, and Quarterly"
+  type        = map(any)
+  default     = { "default" : "Monthly" }
+}
+
+variable "budget_operator" {
+  description = "A choice of operator, options are: EqualTo, GreaterThan, or GreaterThanOrEqualTo"
+  type        = map(any)
+  default     = { "default" = "EqualTo" }
+}
+
+variable "budget_threshold" {
+  type    = map(any)
+  default = { "default" = 80.0 }
```

Comment:
I'm not sure I understand what's happening with these map variables, maybe you could clear this up for me.  What is the `default` key supposed to represent here?

Reply:
I believe I left them in there for myself as to how to use the maps. Took out the defaults and did an apply. Everything worked so I took them out.